client/components/price/price-filter.js

```diff
@@ -0,0 +1,74 @@
+const frenchTouch = {
+  TTCOnly: false,
+  HTOnly: false,
+  withTTC: true,
+};
+
+const usTouch = {
+  TTCOnly: false,
+  HTOnly: true,
+  withTTC: false,
+};
+
+const deutchTouch = {
+  TTCOnly: true,
+  HTOnly: false,
+  withTTC: true,
+};
+
+const showTaxes = {
+  CA: usTouch,
+  WE: usTouch,
+  WS: usTouch,
+  QC: usTouch,
+  DE: deutchTouch,
+  FI: deutchTouch,
+  SN: deutchTouch,
+  CZ: frenchTouch,
+  ES: frenchTouch,
+  FR: frenchTouch,
+  GB: frenchTouch,
+  IE: frenchTouch,
+  IT: frenchTouch,
+  LT: frenchTouch,
+  MA: frenchTouch,
+  NL: frenchTouch,
+  PL: frenchTouch,
+  PT: frenchTouch,
+  TN: frenchTouch,
+};
+
+function priceFilter($translate) {
+  function format(price, country, frequency) {
+    const theCountry = country || 'FR';
+    const taxes = showTaxes[theCountry];
+
+    if (price.withTax.value !== 0) {
+      if (taxes.TTCOnly) {
+        return `<span class="bold">${price.withTax.text}</span>`;
+      }
+      if (taxes.HTOnly) {
+        return `<span class="bold">${price.withoutTax.text}</span>`;
+      }
+      if (frequency === 'yearly') {
+        return `<span class="bold">${$translate.instant('common_price_ht_label', { price: price.withoutTax.text })}`
+                          + `<small>${$translate.instant('common_price_label_yearly')}</small></span> `
+                          + `<span class="italic small"> (${$translate.instant('common_price_ttc_label', { price: price.withTax.text })}`
```

Comment:
You probably moved this code from `ovh-utils-angular` but you can in addition replace some deprecated classes with current helpers we've got (https://getbootstrap.com/docs/4.1/utilities/text/#font-weight-and-italics)